firmware/spiders/asus.py

```diff
@@ -34,7 +35,13 @@ class AsusSpider(Spider):
     ]
 
     def parse(self, response):
-        for url_redirect in set(response.xpath('//div[contains(@class, "ProductCardNormal")]//a/@href').getall()):
+        url_redirects = set()
+        header_scripts = set(response.xpath('//head//script/text()').getall())
+        for header in header_scripts:
+           if not '"url"' in header:
+               continue
+           url_redirects.update(re.findall('"url": "(https://[\w\d\-\_\./]+)"', header))
```

Comment:
```suggestion
            if not '"url"' in header:
                continue
            url_redirects.update(re.findall('"url": "(https://[\w\d\-\_\./]+)"', header))
```
There is mixed indentation here.

Reply:
Urgh. Fixed.